enterprise/frontend/src/embedding-sdk/package.template.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metabase/embedding-sdk-react",
-  "version": "0.55.1-nightly",
+  "version": "0.55.0-metabot",
```

Comment:
This is from https://github.com/metabase/metabase/pull/56281. There was a conflct with the one from master.